modules/jvm/module.yaml

```
@@ -61,3 +61,6 @@ envs:
 - name:         GC_CONTAINER_OPTIONS
   description:  specify Java GC to use. The value of this variable should contain the necessary JRE command-line options to specify the required GC, which will override the default of `-:+UseParallelGC`.
   example:      -XX:+UseG1GC
+- name:         JAVA_COMPACT_OBJECT_HEADERS
+  description:  Whether to enable compact object headers (JEP519) for JDK24 and newer. Enabled by default.
```

```suggestion
  description:  Whether to enable compact object headers (JEP 519) for JDK 25 and newer. Enabled by default.
```

JDK 25 and newer, since its only a product (non-experimental option since then).